### PR TITLE
Liked videos, history, watch later

### PR DESCRIPTION
**Pull request by vivekgudipudi**
image
image
image
image
image

### CODE REVIEW

**Review by Varu98 — APPROVED** [review]
Clean code !

File: src/Styles/Explore.css [hunk 1]
```diff
@@ -75,14 +75,35 @@
...
+.card-video{
+    width: 100%;
+    padding: var(--md);
+}
+.card-video-iconbox{
+    padding: var(--md);
+    font-size: var(--lg);
+}
+.video-action-icon{
+    padding: var(--xs) var(--xl);
```
Comment (parthpandyappp) [inline src/Styles/Explore.css:97]: good use of variables

File: src/components/historyCard.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+export const HistoryCard = ({video})=> {
+    const [showVideoOptions,setShowVideoOptions] = useState(false);
+    const { liked,removeFromLiked,addToLiked } = UseLiked();
+    const { watchLater, removeFromWatchLater, addToWatchLater } = UseWatchLater();
+    const { history, removeFromHistory } = UseHistory();
+
+
+
+    return(
+        <>
```
Comment (parthpandyappp) [inline src/components/historyCard.js]: you can remove this redundant fragment

**Review by parthpandyappp — COMMENTED** [review]
Left few comments, look into'em. Rest Looks good to me, good work on UI :fire:

File: src/components/historyCard.js (hunk 2 above)
Comment (vivekgudipudi) [reply src/components/historyCard.js]: sure, thanks for pointing

File: src/Styles/Explore.css (hunk 1 above)
Comment (vivekgudipudi) [reply src/Styles/Explore.css:97]: thanks

**Review by parthpandyappp — APPROVED** [review]

**Pull request merged**